src/components/blog/BlogContent.module.css

```
@@ -0,0 +1,280 @@
+.container {
+  max-width: 800px;
+  margin: 0 auto;
+  padding: 2rem 1rem;
+}
+
+.backLink {
+  display: inline-flex;
+  align-items: center;
+  gap: 0.5rem;
+  color: #97b753;
+  text-decoration: none;
+  margin: 1rem auto;
+  max-width: 800px;
+  font-size: 0.9rem;
+  font-weight: 500;
+  transition: color 0.2s ease;
+}
+
+.backLink:hover {
+  color: #b3da60;
+}
+
+.header {
+  padding: 1rem 0;
+  margin-bottom: 2rem;
+  border-bottom: 1px solid #e0e0e0;
+}
+
+.title {
+  font-size: 2.5rem;
+  font-weight: bold;
+  margin-bottom: 1rem;
+  color: #333;
+  line-height: 1.2;
+}
+
+.meta {
+  display: flex;
+  align-items: center;
+  gap: 1.5rem;
+  margin: 2rem .5rem;
+  font-size: 1rem;
+  justify-content: space-between;
+  color: #666;
+}
+
+.date {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.author {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+}
+
+.tags {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 0.5rem;
+}
+
+.heroImage {
+  width: 100%;
+  position: relative;
+  margin-bottom: 2rem;
+  border-radius: 8px;
+  overflow: hidden;
+  transition: height 0.4s ease-in-out;
+  cursor: pointer;
+}
+
+.heroImageImg {
+  object-fit: cover;
+}
+
+.tag {
+  background: #f0f0f0;
```

tag のtext 日本語と英語でそろえるにはこうできる
```suggestion
  text-box-trim: trim-both;
```